site/src/atomics/colors.md

```diff
@@ -0,0 +1,229 @@
+---
+title: Color
+description: Color and background-color related atomic css classes for the Atlas Design System
+template: standard
+classType: Atomics
+classPrefixes:
+  - color
+  - background-color
+---
+
+# Color Atomics
+
+Atlas has CSS classes that allow you to easily apply themed background and text colors. Change the site's theme to see how each looks in different themes.
+
+## Accessibility concerns
+
+Altas components take care of ensuring accessible text-to-background color contrast ratios. However, when apply specific colors yourself via atomics always remember to test contrast.
+
+## Text colors
+
+Atlas has three themed text colors.
+
+<table class="table">
+	<caption>
+		Text color classes
+	</caption>
+	<thead>
+		<tr>
+			<th>Text color</th>
+			<th>Text class</th>
+		</tr>
+	</thead>
+	<tbody>
+		<tr class="color-text">
+			<td>Text</td>
+			<td><code>.color-text</code></td>
+		</tr>
+		<tr class="color-text-subtle">
+			<td>Text subtle</td>
+			<td><code>.color-text-subtle</code></td>
+		</tr>
+		<tr class="color-text-invert background-color-alternate">
+			<td>Text invert</td>
+			<td><code>.color-text-invert</code></td>
+		</tr>
+	</tbody>
+</table>
+
+## Background colors
+
+Atlas has five themed (non-sematic) background colors.
+
+<table class="table">
+	<caption>
+		Body color classes
+	</caption>
+	<thead>
+		<tr>
+			<th>Background color</th>
+			<th>Background class</th>
+		</tr>
+	</thead>
+	<tbody>
+		<tr class="background-color-body">
+			<td>Body</td>
+			<td><code>.background-color-body</code></td>
+		</tr>
+		<tr class="background-color-body-medium">
+			<td>Body medium</td>
+			<td><code>.background-color-body-medium</code></td>
+		</tr>
+		<tr class="background-color-body-accent">
+			<td>Body accent</td>
+			<td><code>.background-color-body-accent</code></td>
+		</tr>
+		<tr class="background-color-alternate color-text-invert">
+			<td>Alternate</td>
+			<td><code>.background-color-alternate</code></td>
+		</tr>
+		<tr class="background-color-alternate-medium color-text-invert">
+			<td>Alternate medium </td>
+			<td><code>.background-color-alternate-medium</code></td>
+		</tr>
+	</tbody>
+</table>
+
+## Semantic colors
+
+Semantic colors are themed colors that vary depending on the active theme. They have names that pertain to their usage. Generally speaking, these are the colors used to paint components too.
+
+<table class="table">
+	<caption>
+		Primary color classes
+	</caption>
+	<thead>
+		<tr>
+			<th>Background color /<br>Text color</th>
+			<th>Font class</th>
```

Comment:
Above this point, you've referred to this as "Text class," but below this point, it's "Font class." I think this could be good to unify - I'd prefer "Text class"